src/domain/notifications/index.ts

```diff
@@ -8,3 +10,137 @@ export const NotificationType = {
   OnchainPayment: "onchain_payment",
   LnInvoicePaid: "paid-invoice",
 } as const
+
+export const NotificationChannel = {
+  Push: "push",
+} as const
+
+export const GaloyNotificationCategories = {
+  Payments: "Payments" as NotificationCategory,
```

Comment:
no Circles?

Reply:
These are notification categories that Galoy core knows about. "Circles" will be a NotificationCategory but one that is passed through the graphql api and not one that is known about in our core domain.